### PR TITLE
Added pod selector and added title as props to workload selector

### LINKED ISSUE #970: Related workloads displayed in Network Policies
**Description**
As a user, I want to see related workload from Pod Selector.

### PR DESCRIPTION
**Pull request by mrCherry97**
**Description**
As a user, I want to see connected pods.
Changes proposed in this pull request:
- Added pod selector
- Added title as props to workload selector
- Added title in sidecars workload selector

**Related issue(s)**
Closes #970

### CODE REVIEW

File: core-ui/src/shared/components/WorkloadSelector/WorkloadSelector.js [hunk 1]
```diff
@@ -25,10 +22,7 @@ export const WorkloadSelector = ({ resource, labels }) => {
   return (
     <LayoutPanel className="fd-margin--md" key="workload-selector">
       <LayoutPanel.Header>
-        <LayoutPanel.Head
-          title={t('workload-selector.title')}
-          className="header"
-        />
+        <LayoutPanel.Head title={title} className="header" />
```
Comment (dariadomagala) [inline core-ui/src/shared/components/WorkloadSelector/WorkloadSelector.js:25]: Let's add a default title
```suggestion
        <LayoutPanel.Head title={title || t('workload-selector.title')} className="header" />
```

File: core-ui/src/components/Predefined/Details/Sidecars/Sidecars.details.js [hunk 2]
```diff
@@ -19,6 +19,7 @@ export const SidecarsDetails = ({ DefaultRenderer, ...otherParams }) => {
     <WorkloadSelector
       resource={sidecar}
       labels={sidecar.spec?.workloadSelector?.labels}
+      title={t('workload-selector.title')}
```
Comment (dariadomagala) [inline core-ui/src/components/Predefined/Details/Sidecars/Sidecars.details.js:22]: Once we add a default title in the `WorkloadSelector` component, this line won't be needed

File: core-ui/src/components/Predefined/Details/NetworkPolicy/NetworkPolicy.details.js [hunk 3]
```diff
@@ -60,15 +60,19 @@ export function NetworkPoliciesDetails({ DefaultRenderer, ...otherParams }) {
...
-    Egresses,
-  ];
+  const PodSelector = policy => {
+    const { t } = useTranslation();
+
+    return (
+      <WorkloadSelector
+        resource={policy}
+        labels={policy.spec.podSelector?.matchLabels}
+        title={t('network-policies.headers.pod-selector')}
```
Comment (dariadomagala) [inline core-ui/src/components/Predefined/Details/NetworkPolicy/NetworkPolicy.details.js:70]: why do we use a different title here?

**Review by dariadomagala — APPROVED** [review]

**Review by grego952 — APPROVED** [review]

**Pull request merged**